tools/zos/modifysidedeck.sh

```diff
@@ -12,19 +12,22 @@ originalsidedeck=$1
 outputsidedeck=$2
 newdllname=$3
 
+SCRIPT_DIR=$(dirname $0)
 ID=`date +%C%y%m%d_%H%M%S`
-TMP="/tmp/sidedeck-$(basename $0).$ID.tmp"
-TMP2="/tmp/sidedeck-$(basename $0).$ID.tmp.2"
+TMP="/tmp/sidedeck-$(basename "$0").$ID.tmp"
+TMP2="/tmp/sidedeck-$(basename "$0").$ID.tmp.2"
 
 # Remove on exit/interrupt
-trap "/bin/rm -rf $TMP $TMP2 && exit" EXIT INT TERM QUIT HUP
+trap '/bin/rm -rf "$TMP" "$TMP2" && exit' EXIT INT TERM QUIT HUP
```

Comment:
The double quote inside the single quote won't be interpreted. Is this change needed?

Reply:
yes that was 1 of linter's complain, and I think we do not want it to be interpreted when running this line but instead when trapping one of the signal